### DISCUSSION TITLE
Layout is not able to be edited in playground

### DISCUSSION THREAD

**Discussion by dohomi** [facebook/lexical#6015] [Ideas]
Hey Guys,

love the work on Lexical and especially the playground is a great inspiration to check how to implement custom elements. From the playground perspective the layout seems not to be able to be edited/changed/removed at all. I think thats OK from the demo purpose but not a real world scenario as its unable to remove a layout again from the editor.

Cheers

**Comment by BenJavaCode** [comment]
If you mean the layout node from the layoutPlugin then the layout is removable. The selection is just not working properly so that you have to have a node before the layout to remove it. 
You can create a new command that listens for the KEY_BACKSPACE_COMMAND or the KEY_DELETE_COMMAND with priority set to low.
Then you will need to write the logic that checks if the layout node is part of the selection and then remove it properly. you might also need to stop the event propapagation with something like:     
```
    event.preventDefault();
    event.stopImmediatePropagation();
    event.stopPropagation();
```

Feel free to ask if you need help with it.

**Reply by dohomi** [reply]
Thanks for your reply I will try that. For now its also good enough to use the mouse to mark a layout and then hit the backspace

**Comment by ivailop7** [comment]
Yes, there isn't a UI to edit the already inserted columns layout. The Playground is just a Demo of what could be built with the framework, it's not intended for as-is copy-paste productionisation.
